tfhe/Cargo.toml

```diff
@@ -38,7 +35,6 @@ concrete-csprng = { version = "0.3.0", features = [
     "generator_fallback",
     "parallel",
 ] }
-lazy_static = { version = "1.4.0", optional = true }
```

Comment:
`once_cell` and `lazy_static` are similar. As such, it is desired to only have one of them.